mlflow/java/client/pom.xml

```diff
@@ -17,14 +17,6 @@
     <mlflow.shade.packageName>org.mlflow_project</mlflow.shade.packageName>
   </properties>
 
-  <distributionManagement>
```

Comment:
Do we need to remove the bintray config for now? Or could we leave it around for the next few releases until we're confident the direct upload to Sonatype works?